templates/patterns/list_item_blocks/pattern-list-item-block-one-column.html.twig

```diff
@@ -4,31 +4,14 @@
  * List item block: one column.
  */
 #}
-{% if title %}
-<h3 class="ecl-u-type-heading-3">
-  {{ title }}
-</h3>
-{% endif %}
-
-<ul class="ecl-listing{{ type ? " ecl-listing--" ~ type }}">
-  {{ items }}
-</ul>
-
-{% if button_label and button_url %}
-<div class="ecl-u-clearfix ecl-u-pv-m">
-  {% include '@ecl/link' with {
-    'link': {
-      'type': 'standalone',
-      'path': button_url,
-      'label': button_label
-    },
-    'icon': {
-      'path': ecl_icon_path,
-      'type': 'ui',
-      'name': 'fullscreen',
-      'size': 's',
-    },
-    'extra_classes': 'ecl-u-f-r',
-  } %}
-</div>
-{% endif %}
+{% include '@oe_theme/compositions/ec-component-content-item-block/content-item-block.html.twig' with {
+  content_item_block : {
+    type: type|default('one'),
+    title: title,
+    items: items,
+    button: {
+      path: button_url,
+      label: button_label
+    }
+  }
```

Comment:
As said above, let's make this simpler by just passing the inner properties, without wrapping them into `content_item_block`. Also we just do `type: 'one',`.

Reply:
I removed the `content_item_block`, but I don't remove `type: 'one',` because this template is one two and three columns.

Reply:
Yeah but we are in the `-one-colum.html.twig` template, we set this value here as `one`, it cannot come from outside.

Reply:
I changed the template, according to our conversation.